### PR TITLE
Fix missing CPU memory info in the instance type VM create details section

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

This bug is happening because we rely on 4 constant serieses instead of reading data from the IT resource.

## 🎥 Demo
Before:
CPU-memory-exists-b4
After:
CPU-memory-exists

### CODE REVIEW

File: src/utils/hooks/useKubevirtTranslation.ts [hunk 1]
```diff
@@ -10,4 +10,4 @@ export const useKubevirtTranslation = () => useTranslation('plugin__kubevirt-plu
  * @param value string to translate
  */
 // skipcq: JS-C1002
-export const t = (value: string) => getI18n().t(value, { ns: 'plugin__kubevirt-plugin' });
+export const t: TFunction<'plugin__kubevirt-plugin', undefined> = getI18n().t;
```
Comment (metalice) [inline src/utils/hooks/useKubevirtTranslation.ts:13]: why remove the namespace? what is the default name space?

Comment (avivtur) [reply src/utils/hooks/useKubevirtTranslation.ts:13]: the namespace is declared in the type, I changed it so we can also send translations like these:
`"{{cpu}} CPUS", { cpu: some_value }"`

**Pull request merged**